jetson/nav/README.md

```diff
@@ -1,156 +1,203 @@
+
+To run the Navigation Simulator, you will need to open at least 3 terminals from the `mrover-workspace` directory (at
+the top level). Run `vagrant up` in one of the terminals, and once that completes, run `vagrant ssh` in all three of
+them. You will then follow these commands in order, one set of commands for each terminal. Once the three sets of
+commands are run and in use, go to the [Navigation Simulator Website](http://localhost:8010/). Note, this will not
+show anything until the last set of commands are run.
 
 ### Terminal 1 (`jetson/nav` code)
-Run the command `$./jarvis build jetson/nav`, this command will compile all the navigation directory code. If it compiles succecssful, no errors will be returned. Then, run `$./jarvis exec jetson/nav` to start the navigation code.
+
+Run the command `./jarvis build jetson/nav`, this command will compile all the navigation directory code. If it
+compiles successful, no errors will be returned. Then, run `./jarvis exec jetson/nav` to start the navigation code.
 
 ### Terminal 2 (LCMs)
-Run the command `$./jarvis build lcm_bridge/server`, this command will build the LCM messages. If it compiles successful, no errors will be returned. Then, run `$./jarvis exec lcm_bridge/server` to run the LCM messages.
+
+Run the command `./jarvis build lcm_bridge/server`, this command will build the LCM messages. If it compiles
+successful, no errors will be returned. Then, run `./jarvis exec lcm_bridge/server` to run the LCM messages.
 
 ### Terminal 3 (Navigation Simulator)
-Run the command `$./jarvis build simulators/nav`, this command will build the Navigation Simulator. If it compiles successful, no errors will be returned. Then, run `$./jarvis exec simulators/nav` to run the Simulator.
 
-If desired, you can run a fourth terminal for debugging purposes via LCM messages. To do so, make sure you have another terminal, and starting in the `mrover-workspace` directory, run `vagrant ssh`. Once we are ssh'ed into the virtual machine, run `$./jarvis build lcm_tools/echo` to build the echo tool for LCMs. This will return the messages that are being communicated between publishers and subscribers. To run, enter the command `$./jarvis exec lcm_tools/echo TYPE_NAME CHANNEL` to echo the specified LCM and channel. (These are described in our LCM section and ICDs on the Drive)
+Run the command `./jarvis build simulators/nav`, this command will build the Navigation Simulator. If it compiles
+successful, no errors will be returned. Then, run `./jarvis exec simulators/nav` to run the Simulator.
 
+If desired, you can run a fourth terminal for debugging purposes via LCM messages. To do so, make sure you have another
+terminal, and starting in the `mrover-workspace` directory, run `vagrant ssh`. Once we are ssh'ed into the virtual
```

Comment:
i would just link the esw guide to this https://github.com/umrover/embedded-testbench/wiki/LCM-Usage

Reply:
kk good to consolidate resources